src/fusor/translator.py

```diff
@@ -180,6 +193,42 @@ def _get_genomic_ac(self, chrom: str, build: Assembly) -> str:
             raise ValueError
         return alias_list[0].split(":")[1]
 
+    def _assess_gene_symbol(
+        self, gene: str, caller: Caller
+    ) -> list[GeneElement | UnknownGeneElement, str]:
+        """Determine if a gene symbol exists and return the corresponding
+        GeneElement
+
+        :param gene: The gene symbol
+        :param caller: The gene fusion caller
+        :return A list containing a GeneElement or UnknownGeneElement and a string
+        """
+        if gene == "NA":
+            return UnknownGeneElement(), "NA"
+        return self._get_gene_element(gene, caller), self._get_gene_element(
+            gene, caller
+        ).gene.label
+
+    def _process_gene_symbols(
+        self, gene_5prime: str, gene_3prime: str, caller: Caller
+    ) -> dict[GeneElement | UnknownGeneElement]:
```

Comment:
Update docstring too
```suggestion
    ) -> GeneFusionPartners:
```